Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -348,8 +324,7 @@ u32 Read_Opcode(u32 _Address)
 		return 0x00000000;
 	}
 
-	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bMMU &&
-		(_Address & ADDR_MASK_MEM1))
+	if ((_Address & ADDR_MASK_MEM1))
```

Comment:
@magumagu I believe that's only when there's a bitwise OR present. e.g. Something like `(val & 0xFFFF | val >> 16)`